frontend/src/components/Footer.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="footer p-3">
-    <h3><a href="http://codeforpoznan.pl">Code for Poznan</a></h3>
+    <h3 class="h3"><a href="http://codeforpoznan.pl">Code for Poznan</a></h3>
```

Comment:
That seems a bit too much, why not style the h3 element?

Reply:
Using class names is always better idea. Gives more flexibility. Perhaps the name its-self is not that fortunate. I will change it to heading